app/assets/stylesheets/pages/devise/_new_reg.scss

```diff
@@ -0,0 +1,20 @@
+.background {
```

Comment:
may be we could have a more descriptive class ? background is a bit too general and could be confusing